server.js

```diff
@@ -3,26 +3,43 @@ import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
+import netflixData from './data/netflix-titles.json'
+
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/books"
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
+const Director = mongoose.model('Director', {
+  name: String,
+})
+
+const Title = mongoose.model('Title', {
+  title: String,
+
+  director: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'Director'
+  }
+})
```

Comment:
I think you're off to a good start, but I'm not sure this is 100% the right way to do it. From what I've understood, the "Model" (in your case the Director and the Title) should rather be a model of the data in your JSON file. 

And in the Netflix json data file, it's all listing "Shows", sort of. 

It's confusing and complicated, I know, but maybe something like this:

```
const Show = new mongoose.model('Show', {
  show_id: Number,
  title: String,
  director: String,
  cast: String,
  country: String,
  date_added: String,
  release_year: Number,
  rating: String,
  duration: String,
  listed_in: String,
  description: String,
  type: String,
})
```